Vagrantfile

```diff
@@ -63,6 +63,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       mount_options: synced_folder.include?('mount_options') ? synced_folder['mount_options'] : []
   end
 
+  # Provision using file provisioner to add known_hosts from host to guest.
+  config.vm.provision "file", source: "~/.ssh/known_hosts", destination: "~/.ssh/known_hosts"
```

Comment:
@geerlingguy you are right, I haven't test it on Windows and I can guess it won't work on it. I can move it to an ansible task just tell me where you prefer to place it then I can check if the file exists or not.